src/views/Operations/ServerPowerOperations/ServerPowerOperations.vue

```diff
@@ -272,17 +278,34 @@ export default {
           this.$bvModal
             .msgBoxConfirm(modalMessage, modalOptions)
             .then((confirmed) => {
-              if (confirmed) this.$store.dispatch('controls/serverSoftReboot');
+              if (confirmed) {
+                if (!this.isReadOnlyUser) {
```

Comment:
For read only user reboot option is available?

Reply:
Reboot is not available for read-only user,
Also we don't show the pop-up for read-only user. Hence this condition is added.

Reply:
So if reboot is not available for read-only then we can remove it ?

Reply:
It can be done, But previously it is that, If we are read-only user and clicks on reboot. An error message pops up